addons/sys_intercom/XEH_postInit.sqf

```diff
@@ -13,7 +13,6 @@ if (!hasInterface) exitWith {};
 ["Tank", "init", FUNC(passengerIntercomAction), nil, nil, true] call CBA_fnc_addClassEventHandler;
 ["Car_F", "init", FUNC(passengerIntercomAction), nil, nil, true] call CBA_fnc_addClassEventHandler;
 
-
```

Comment:
2 empty lines here are to clearly separate the stable code and the developer/debug code.